presto-hive/src/test/java/io/prestosql/plugin/hive/s3/MockAmazonS3.java

```diff
@@ -125,6 +131,14 @@ public ListObjectsV2Result listObjectsV2(ListObjectsV2Request listObjectsV2Reque
             listingV2.getObjectSummaries().add(standardOne);
             listingV2.setTruncated(true);
             listingV2.setNextContinuationToken(continuationToken);
+
+            if (hasHadoopEmptyFolderObjects) {
+                S3ObjectSummary glacier = new S3ObjectSummary();
```

Comment:
Probably copy/pasta, but can you use a different variable name?

Reply:
Thanks for catching that